app/src/main/java/com/cornellappdev/transit/models/Stop.kt

```diff
@@ -5,6 +5,16 @@ import com.squareup.moshi.Json
 /**
  * Data class representing a TCAT stop
  */
+
+enum class Type{
+    BUSSTOP{
+
+    },
+    PLACE{
+        
+    }
+}
```

Comment:
uhhhhhhhhhh did you mean to commit this 🤣

Reply:
oh shoot I started working on it but never finished my bad 😭 Lemme fix real quick